R/summary.R

```diff
@@ -440,7 +440,7 @@ summary.Graph = function(object, resample_result = NULL, control = summary_contr
 #' @param importance_measures (character()|NULL)\cr
 #'   vector of importance measure names. Possible choices are "pfi.<loss>"
 #'   ([iml::FeatureImp]), "pdp" ([iml::FeatureEffects], see ) and
-#'   "shap" ([fastshap::explain]). Default of NULL results in "pfi.<loss> and
```

Comment:
The "<loss>" does not render for me, do you know why?

Reply:
Haha, also not in my comment :) it is about the smaller-than-loss-greater-than signs

Reply:
Thanks. I know how to fix this :-) a ` must be used